### PR TITLE
changes for  rack manager

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[changes for  rack managers]

### Issues Resolved
[]

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/rack_managers.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	"fmt"
+
+	"github.com/HewlettPackard/oneview-golang/ov"
+)
+
+func main() {
+	var (
+		ClientOV        *ov.OVClient
+		RackManagerName = "5UF7201002" //"TestRackManagerGOsdk"
+		// new_fc_name = "RenamedFCNetwork"
```
Comment (AvijitOS) [inline examples/rack_managers.go:13]: remove commented section

File: examples/rack_managers.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+		config.OVCred.UserName,
+		config.OVCred.Password,
+		config.OVCred.Domain,
+		config.OVCred.Endpoint,
+		config.OVCred.SslVerify,
+		config.OVCred.ApiVersion,
+		config.OVCred.IfMatch)
+
+	// scope := ov.Scope{Name: "ScopTest", Description: "Test from script", Type: "ScopeV3"}
+	// _ = ovc.CreateScope(scope)
```
Comment (AvijitOS) [inline examples/rack_managers.go:30]: remove commented section

File: examples/rack_managers.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+	rmcName := rmCreated.Name
+
+	err = ovc.DeleteRackManager(rmcName)
+	if err != nil {
+		panic(err)
+	} else {
+		fmt.Println("Deleted Rack manager successfully...")
+	}
+
+	// fcNetwork01 := ov.FCNetwork{
```
Comment (AvijitOS) [inline examples/rack_managers.go:77]: remove commented section

File: ov/rack_manager.go [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+	if err := json.Unmarshal([]byte(data), &rms); err != nil {
+		return rms, err
+	}
+	return rms, nil
+}
+
+// Add rack manager- Add  new rack manager
+func (c *OVClient) AddRackManager(rm RackManager) (resourceId string, err error) {
+
+	log.Infof("Initializing adding of RackManager %s.", rm.Hostname)
```
Comment (AvijitOS) [inline ov/rack_manager.go:177]: do we need this info as we are adding debug log mostly

File: ov/rack_manager.go [hunk 5]
```diff
@@ -0,0 +1,261 @@
+/*
+(c) Copyright [2021] Hewlett Packard Enterprise Development LP
```
Comment (AvijitOS) [inline ov/rack_manager.go:2]: change the copy right year to 2022

Comment (nabhajit-ray) [reply ov/rack_manager.go]: Done

File: ov/rack_manager.go (hunk 4 above)
Comment (nabhajit-ray) [reply ov/rack_manager.go:176]: Done

Comment (nabhajit-ray) [reply ov/rack_manager.go]: Done

File: examples/rack_managers.go (hunk 1 above)
Comment (nabhajit-ray) [reply examples/rack_managers.go]: Done

**Review by AvijitOS — APPROVED** [review]

File: ov/config.go [hunk 6]
```diff
@@ -26,10 +86,15 @@ func LoadConfigFile(configFile string) (Configuration, error) {
 	defer configF.Close()
 	if err != nil {
 		fmt.Println(err)
+		fmt.Println("error opening json file")
 		return config, err
 	}
 	jsonParser := json.NewDecoder(configF)
-	jsonParser.Decode(&config)
+	err_unmarhsal := jsonParser.Decode(&config)
```
Comment (alisha-k-kalladassery) [inline ov/config.go:93]: unmarhsal typo

Comment (nabhajit-ray) [reply ov/config.go]: done

**Review by alisha-k-kalladassery — APPROVED** [review]

**Pull request merged**